### PR TITLE
Add launch.json

### PR DESCRIPTION
**Pull request by pmconne**
Copied directly from old repo, removed Visualization tests.

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,586 @@
...
+            "name": "(Windows) Attach",
+            "type": "cppvsdbg",
+            "request": "attach",
+            "processId": "${command:pickProcess}"
+        },
+        {
+            "name": "LNX Native BeHttpTest",
+            "type": "cppdbg",
+            "request": "launch",
+            "program": "${env:OutRoot}LinuxX64/static/ProductCopy/BeHttp-Gtest/BeHttpTest",
```
Comment (calebmshafer) [inline .vscode/launch.json:160]: We should remove all of the dlls that no longer exist.

Comment (pmconne) [reply .vscode/launch.json]: Done.

**Review by calebmshafer — APPROVED** [review]

**Comment by pmconne** [comment]
Linux failed with the following, nothing to do with this PR, something to do with @khanaffan's #27 which isn't even merged yet...
```
  1) concurrent query tests
       reset config:
     TypeError: conn.concurrentQueryResetConfig is not a function
      at Function.resetConfig (lib/test/ECDb.test.js:50:21)
      at Context.<anonymous> (lib/test/ECDb.test.js:65:51)

  2) concurrent query tests
       query timeout:
     TypeError: conn.concurrentQueryResetConfig is not a function
      at Function.resetConfig (lib/test/ECDb.test.js:50:21)
      at Context.<anonymous> (lib/test/ECDb.test.js:98:31)

  3) concurrent query tests
       query max memory:
     TypeError: conn.concurrentQueryResetConfig is not a function
      at Function.resetConfig (lib/test/ECDb.test.js:50:21)
      at Context.<anonymous> (lib/test/ECDb.test.js:108:31)

  4) concurrent query tests
       restart query:
     TypeError: conn.concurrentQueryResetConfig is not a function
      at Function.resetConfig (lib/test/ECDb.test.js:50:21)
      at Context.<anonymous> (lib/test/ECDb.test.js:118:31)
```

**Comment by calebmshafer** [comment]
I can merge this but very concerned about how @khanaffan 's changes potentially made it into the build.

@chuckkir any idea?

**Comment by chuckkir** [comment]
The JS build puts output into the source tree. The problem file is iModelJsNodeAddon\api_package\ts\lib\test\ECDb.test.js which is not part of the source so it doesn't get updated on a pull but also is not deleted. Is it possible to move it to the output tree? If not, I guess we need a pipeline step to purge the bits of source that aren't actually source (lib on down in this case).

**Pull request merged**